public/templates/directives/notification-list.html

```diff
@@ -1,7 +1,14 @@
 <div class="notif-list">
-    <div ng-repeat="notification in notifications" class="notification" ng-class="{'unread': !notification.is_read}">
-        <a href="{{ ::notification.url }}" class="img-link"><img pfm-src-loader="::notification.thumbnail_url" pfm-src-size="thumbnail"></a>
-        <a href="{{ ::notification.url }}" class="message"><p>{{ ::notification.text }}</p></a>
+    <div class="notif-switch" ng-hide="switchHidden">
+        <input id="test" type="checkbox" hidden="hidden" ng-model="subscribed" ng-change="switchToggled()" ng-disabled="switchDisabled"/>
+        <label for="test" class="switch" ng-class="{'disabled': switchDisabled}"></label>
+        <span>Enable push notifications</span>
+    </div>
+    <div class="notif-scroll">
+        <div ng-repeat="notification in notifications" class="notification" ng-class="{'unread': !notification.is_read}">
+            <a href="{{ ::notification.url }}" class="img-link"><img pfm-src-loader="::notification.thumbnail_url" pfm-src-size="thumbnail"></a>
+            <a href="{{ ::notification.url }}" class="message"><p>{{ ::notification.text }}</p></a>
+        </div>
+        <p ng-show="notifications.length < 1" class="error">No notifications :(</p>
```

Comment:
If an element isn't getting shown and hidden very often, `ng-if` should be preferred over `ng-show` as it avoids spawning the DOM element if it shouldn't be shown (`ng-show` spawns it and hides it with CSS if needed).